templates/community-viz/scripts/dscc-scripts.js

```diff
@@ -17,37 +17,62 @@
  * limitations under the License.
  */
 
-const args = process.argv.slice(2);
+const argparse = require('argparse');
 const shell = require('shelljs');
 const chalk = require('chalk');
-const script = args[0];
 
 const msg = require('./bin/message.js');
 const push = require('./bin/push.js');
 const build = require('./bin/build.js');
 
-switch (script) {
+const parser = new argparse.ArgumentParser({
+  version: '1.0',
+  addHelp: true,
+  description: 'Scripts to manage viz dev and deployment',
+});
+
+parser.addArgument(['-s', '--script'], {
+  choices: ['start', 'build', 'push', 'updateMessage'],
+  dest: 'script',
+  help: 'The script to run.',
+  required: true,
+});
+
+parser.addArgument(['-p', '--prod'], {
+  choices: ['false', 'true'],
+  dest: 'prod',
+  help:
+    'Whether or not to use production options. Only required for the build and push scripts.',
+  required: false,
+});
+
+parser.addArgument(['-f', '--format'], {
+  choices: ['object', 'table'],
+  dest: 'format',
+  help: 'The format for the data. Only required for the updateMessage script.',
+  required: false,
+});
+
+const args = parser.parseArgs();
+
+switch (args.script) {
```

Comment:
can you wrap this switch into a function called main, and then just call it at the end of the file?